packages/alfa-dom/src/node/element.ts

```diff
@@ -352,11 +352,7 @@ export namespace Element {
     );
   }
 
-  export const hasName = predicate.hasName;
-
-  export const hasNamespace = predicate.hasNamespace;
-
-  export const hasId = predicate.hasId;
+  export const { hasId, hasName, hasNamespace, hasTabIndex, isDisabled } = predicate;
```

Comment:
Is there a way to re-export everything inside a namespace?

Reply:
I'm afraid not 😕